tests/tester_db_connector.py

```diff
@@ -1,73 +1,31 @@
+    port = db_config.port
+    host = db_config.host
     protocol = 'json'
     query = 'select * from adrian_test_db;'
 
     # Create pandas DataFrame
     df = pd.DataFrame(data=list(range(30)), index=pd.date_range(start='2014-11-16', periods=30, freq='H'), columns=['0'])
 
-    def test_connect_to_db_remote(self):
-        print("*********Test 1: test_connect_to_db_remote*********")
-        try:
-            print("Create DB Connector Instance")
-            test_db_connector_1 = InfluxDBConnector(self.username, self.password, self.port, self.database, self.host)
-            print("Create test database")
-            test_db_connector_1.create_database(self.database)
-            dict_test = {'name': self.database}
-            dbs = test_db_connector_1.list_databases()
-            self.assertTrue(dict_test in dbs)  # should be in the list
-            print("Write DataFrame")
-            self.assertTrue(test_db_connector_1.write_points(self.df, 'adrian_test_db', protocol=self.protocol))
-            print("Read DataFrame")
-            result = test_db_connector_1.query(self.query)
-            self.assertTrue(result)
-            print("Delete test database")
-            test_db_connector_1.delete_database(self.database)
-            dbs = test_db_connector_1.list_databases()
-            self.assertTrue(dict_test not in dbs)  # should not be in the list
-            print("\ntest_connect_to_db_remote PASSED")
-            print("***************************************************")
-        except ConnectionError as error:
-            self.fail("Test: Failed - {0}\n".format(error))
-"""
-    # will put on hold for now until figure out the way to use local InfluxDB on Travis
-    def connect_to_db_local(self):
-        print("Test 2: test_db_connector_2 - local")
-        try:
-            print("Create DB Connector Instance")
-            test_db_connector_2 = InfluxDBConnector(database=self.database)
-            print("Create test database")
-            test_db_connector_2.create_database(self.database)
-            dict_test = {'name': self.database}
-            dbs = test_db_connector_2.list_databases()
-            self.assertTrue(dict_test in dbs)  # should be in the list
-            print("Write DataFrame")
-            self.assertTrue(test_db_connector_2.write_points(self.df, 'adrian_test_db', protocol=self.protocol))
-            print("Read DataFrame")
-            result = test_db_connector_2.query(self.query)
-            self.assertTrue(result)
-            print("Delete test database")
-            test_db_connector_2.delete_database(self.database)
-            dbs = test_db_connector_2.list_databases()
-            self.assertTrue(dict_test not in dbs)  # should not be in the list
-            print("\ntest_connect_to_db_remote PASSED")
-            print("***************************************************")
-        except ConnectionError as error:
-            print("Test: Failed - {0}\n".format(error))
-"""
```

Comment:
Sorry to be annoying, haha. Please add failure messages for the tests

Reply:
In the beginning I though it will be good to have those, but it seems that Coveralls treats those lines as misses since when successful the test never reaches them. You think we should keep those?

Reply:
Yeah. I think we should keep them

Reply:
ok